static/src/javascripts/projects/common/modules/commercial/epic/iframe-epic-utils.js

```diff
@@ -24,112 +27,97 @@ const OPTIMIZE_EPIC_CHANNEL = 'OPTIMIZE_EPIC';
 const EPIC_INITIALIZED = 'EPIC_INITIALIZED';
 const EPIC_HEIGHT = 'EPIC_HEIGHT';
 
-// outgoing event types
-const FONTS = 'FONTS';
+const createEpicIframe = (url: string): Error | IframeEpicComponent => {
+    let origin;
+    try {
+        origin = new URL(url).origin;
+    } catch (error) {
+        return new Error(`unable to get origin of iframe Epic - ${error}`);
```

Comment:
should we report this error to sentry? this is a bad one since we'll never even insert the iframe!